data/cncf.yaml

```diff
@@ -1735,9 +1735,6 @@
       url: https://github.com/notaryproject/notation
       check_sets:
         - code-lite
```

Comment:
Could you please remove the `code-lite` check set from the `notation` repository? `code-lite` is a actually a subset of `code`, so both shouldn't be assigned to the same repository 🙂

Reply:
oh ok...will make the changes right away